src/components/editor/editor-window/key-map.ts

```diff
@@ -0,0 +1,109 @@
+import CodeMirror, { KeyMap } from 'codemirror'
+import { wrapTextWith } from './tool-bar/utils'
+
+// const isMac = CodeMirror.keyMap.default === CodeMirror.keyMap.macDefault
+
+export const defaultKeyMap: KeyMap = {
+  F10: function (cm) {
+    cm.setOption('fullScreen', !cm.getOption('fullScreen'))
+  },
+  Esc: function (cm) {
+    if (cm.getOption('fullScreen') && !(cm.getOption('keyMap')?.substr(0, 3) === 'vim')) {
+      cm.setOption('fullScreen', false)
+    } else {
+      return CodeMirror.Pass
+    }
+  },
+  'Cmd-S': function () {
+    return CodeMirror.Pass
+  },
+  'Ctrl-S': function () {
+    return CodeMirror.Pass
```

Comment:
This just lets the keybinding pass forward to the browser and does not prevent the saving dialog. Same applies to the `Cmd-S` shortcut.

Reply:
fixed